### ISSUE TITLE
Loss function and Optimizer

### ISSUE THREAD

**Issue by jayavanth** [keunwoochoi/music-auto_tagging-keras#8]
What kind of loss function and what kind of optimizer are you using for this? I couldn't find them in the code. According to the paper, it is cross entropy and adam resp.

**Comment by keunwoochoi** [comment]
That's correct, cross entropy and adam.

**Comment by jayavanth** [comment]
I didn't see any lines in the code referring to the loss function or optimizer. May I know where in the code you're using those? Thanks.

**Comment by keunwoochoi** [comment]
Hm, there's no lines for those in this repo. Adding them in Keras is trivial.
